cmake/OpenCVFindWebP.cmake

```diff
@@ -10,8 +10,6 @@
 
 # Look for the header file.
 
-unset(WEBP_FOUND)
```

Comment:
do not unset that if you want the CI to pass.

Reply:
i think Webp version of buildbot should be updated.